ports/blend2d/vcpkg.json

```
@@ -18,41 +17,14 @@
     }
   ],
   "default-features": [
-    "jit",
-    "logging",
-    "tls"
+    "jit"
```

To fix default installations on arm32:
```suggestion
    {
      "name": "jit",
      "platform": "!arm32"
    }
```